examples/tensorflow/README.md

```diff
@@ -9,22 +9,28 @@ tensorflow/
 ├── README.md
 ├── objects
 │   └── image.png
-├── cnn_mnist.py
+├── mnist.py ==> this is working with tensorflow version >= 2.2
+├── cnn_mnist.py ==> this was working with tensorflow version < 2 
 ├── main.py
 └── requirements.txt
 ```
 
 ### Instructions
 
-Tensorflow only supports python 3.6-3.8 - so make sure you're on a version in that range.
+Tensorflow only supports python >= 3.6 - so make sure you're on a version in that range.
+
+Tensorflow 2 also need install graphviz which is included in file promote.sh. To manually install graphviz, please run:
```

Comment:
well - when you're training the model locally you also need it - so include the link to the install instructions (most people aren't using yum locally)